bte/tasks.py

```diff
@@ -0,0 +1,556 @@
+import asyncio
+import base64
+import io
+import os
+import subprocess
+from tempfile import NamedTemporaryFile
+from typing import Any, AnyStr, ByteString, Dict, List
+
+import eyed3
+import magic
+import requests
+from django.utils.encoding import force_bytes
+from eyed3 import id3
+from lxml.etree import XMLSyntaxError
+from lxml.html.clean import Cleaner
+from PIL.Image import Image
+from PyPDF2 import PdfFileMerger, PdfFileReader
+from PyPDF2.utils import PdfReadError
+from seal_rookery import seals_data, seals_root
```

Comment:
I thought seals were removed?

Reply:
So depends on what you are referring to.  Seal Rookery is on the list for an update.  

I dropped Seal Rookery as a separate stand alone image.  Because it doesnt take that long to build in reality.  And now we have plans to modify it and judge-pics.

Reply:
Sounds good.